plugins/base/build.gradle.kts

```diff
@@ -4,6 +4,8 @@ plugins {
     id("com.jfrog.bintray")
 }
 
+val testUtils by configurations.creating
```

Comment:
What is the reason for this?

Reply:
It allows us to share `test` sources for other modules. It's the simpliest way to have access to abstract classes and `TestOutputWriter`, which are defined in `test` sources, in GFM without any major changes.  Checkout this line
https://github.com/Kotlin/dokka/blob/05db29f3150aa23cdf48b4cefcb3024e60c130d6/plugins/gfm/build.gradle.kts#L13

Reply:
Why can't this be moved into `:test-tools` or `:testApi`? 🤔

Reply:
The problem is they are strongly coupled. There would be circular dependency between modules.

Reply:
The biggest problem is `TestOutputWriter`, as it is `base` EP, so it has to be aware of `base`.